src/app/a11y/OverscrollBg.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from "react";
+import useForceUpdate from "use-force-update";
+
+type OverscrollBgProps = {
+  topClassName: string;
+  bottomClassName: string;
+};
+
+const doc = document.documentElement;
+
+const OverscrollBg: React.FC<OverscrollBgProps> = ({
+  topClassName,
+  bottomClassName
+}) => {
+  const scrollTop = useScrollTop();
+
+  React.useEffect(() => {
```

Comment:
Just curious, shouldn't we use `useLayoutEffect` instead? ([*reference*](https://reactjs.org/docs/hooks-reference.html#uselayouteffect))

Reply:
Oh yeah! We need `useLayoutEffect ` here.